docs/updating.md

```diff
@@ -208,3 +208,26 @@ your answers from the last update.
 Of course, the experience will be less satisfactory. The new template will override any
 changes found in your local project. But you can use a Git diff tool to overcome that.
 After doing this, further updates generally keep on working as usual.
+
+### Aborting an update
+
+When you're not happy with the result of a `copier update` run or unsure about
+adding the introduced changes to your code base, specifically when you have
+unpleasant conflicts, it's not 100% obvious how to get back to the previously
+clean copy of your branch. The following strategies won't work:
+
+-   `git checkout <branch>` – error: you need to resolve your current index first
+-   `git checkout .` – error: path '&lt;filename&gt;' is unmerged
```

Comment:
@sisp It looks like escaping HTML-like tags with backslashes doesn't work:

![Screenshot from 2024-06-21 15-42-42](https://github.com/copier-org/copier/assets/665072/d5f4d825-5c57-4dec-9fbe-4b3cd72569ef)

To my knowledge, HTML entities work as expected, because Markdown accepts all HTML just "as is".

Reply:
I opened #1671 to fix the issue.

Reply:
Oh, sorry about this! I thought this worked, but it obviously doesn't. Interestingly, GFM and GitLab Markdown don't require any special care at all. Anyway, thanks for catching my error! :ok_hand: